src/components/Background/index.css

```diff
@@ -17,41 +19,77 @@
   }
 }
 
-.background-image::after {
-  content: "";
+.image {
+  position: absolute !important;
+
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  z-index: -2;
+
+  width: 100%;
+  height: 100%;
+
+  opacity: 0.9;
+}
+
+.video {
+  position: absolute;
+
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  z-index: -2;
+
+  width: 100%;
+  height: 100%;
 
+  opacity: 0.9;
+
+  object-fit: cover;
+}
+
+.rect {
   position: absolute;
 
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
 
+  z-index: -1;
+
   width: 100%;
   height: 100%;
+
+  background-color: $lavender;
 }
 
-.blend-normal::after {
+.blend-normal {
   mix-blend-mode: normal;
 
   @supports not (mix-blend-mode: difference) {
     background-color: color($fallback-overlay alpha(40%));
   }
 }
 
-.blend-difference::after {
+.blend-difference {
   mix-blend-mode: difference;
 
   @supports not (mix-blend-mode: difference) {
     background-color: color($fallback-overlay alpha(40%));
   }
 }
 
-.color-lavender::after {
+.color-lavender {
   background-color: color($lavender alpha(84%));
 }
 
-.color-light-blue::after {
+.color-light-blue {
```

Comment:
Also remove the ::after for the color-magenta

Reply:
👍